### PR TITLE
Initial checkin for MySQL Monitors.

### PR DESCRIPTION
**Pull request by npande**

### CODE REVIEW

File: monitor_packages/mysql/.terraform.lock.hcl [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# This file is maintained automatically by "terraform init".
```
Comment (arunpatyal) [inline monitor_packages/mysql/.terraform.lock.hcl:1]: This file should not be checked in.

File: monitor_packages/mysql/README.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+# sumologic-mysql-monitors
+
+This Terraform script configures pre-packaged Sumo Logic Monitors for MySQL using Terraform modules.
+For installation and configuration, please look at Sumo Logic MySQL help document.
```
Comment (arunpatyal) [inline monitor_packages/mysql/README.md:4]: Add the help page link. It is going to be this link, if we are updating the existing app and doc pages: https://help.sumologic.com/07Sumo-Logic-Apps/12Databases/MySQL/MySQL-App-Dashboards

File: monitor_packages/mysql/mysql.json [hunk 3]
```diff
@@ -0,0 +1,443 @@
...
+  "children": [
+    {
+      "name": "MySQL - Large number of slow queries",
+      "description": "This alert fires when we detect that there are 5 or more slow queries within a 5 minute time interval.",
+      "type": "MonitorsLibraryMonitorExport",
+      "monitorType": "Metrics",
+      "queries": [
+        {
+          "rowId": "A",
+          "query": "$$mysql_data_source metric=mysql_slow_queries db_cluster=* server=* | sum by db_cluster, server"
```
Comment (arunpatyal) [inline monitor_packages/mysql/mysql.json:14]: Make sure to mention replacing the placeholder value in docs, otherwise, monitors will fail.

**Review by arunpatyal — CHANGES_REQUESTED** [review]

File: monitor_packages/mysql/mysql.json (hunk 3 above)
Comment (npande) [reply monitor_packages/mysql/mysql.json:14]: Doc contains section detailing instruction to perform find and replace with json import based monitors.

File: monitor_packages/mysql/README.md (hunk 2 above)
Comment (npande) [reply monitor_packages/mysql/README.md]: done

File: monitor_packages/mysql/.terraform.lock.hcl (hunk 1 above)
Comment (npande) [reply monitor_packages/mysql/.terraform.lock.hcl]: done

**Comment by arunpatyal** [comment]
LGTM

**Pull request merged**